plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -43,23 +46,21 @@ public void setUp() {
     doc = pluginXmlDocument.get();
   }
 
-  protected Document getDocument() {
+  protected final Document getDocument() {
     return doc;
-  }
-  
-  // Generic tests that should be true of all plugin.xml files
+  } 
   
   @Test
-  public void testRootElementIsPlugin() {
+  public final void testRootElementIsPlugin() {
```

Comment:
No, it isn't; but in this particular case since this is a base class I wanted to make sure these tests didn't get turned off or revised; at least not without everyone realizing that's what they were doing.